storage/innobase/include/log0log.h

```diff
@@ -139,7 +139,12 @@ Closes the log.
 @return lsn */
 lsn_t
 log_close(void);
-/*===========*/
+
+/************************************************************//**
```

Comment:
/** Enables redo ... */
fix only if you have to re-push/re-test the PR otherwise